### ISSUE TITLE
Cannot destructure property 'name' of 'signal' as it is null

### ISSUE THREAD

**Issue by ketan1829** [joaomirandasa/whats2api#67]
======= 2021-02-26 09:42:46: F.install('module', '') ---> TypeError: Cannot destructure property 'name' of 'signal' as it is null. TypeError: Cannot destructure property 'name' of 'signal' as it is null.

**Comment by impozzible** [comment]
Yeah was getting this after updating to the new version of open-wa, the temp fix I have applied is 
1. Goto human-signals/build/src/main.js
2. Comment out line 31-34
```
// const signalsA=Array.from({length},(value,number)=>
// getSignalByNumber(number,signals));

// return Object.assign({},...signalsA);
```

**Comment by impozzible** [comment]
That will fix the error and allow your instance to start

**Comment by manojsitapara** [comment]
@impozzible any permanent fix? how can I make this fix in docker?
